packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/Options.tsx

```diff
@@ -0,0 +1,179 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { FiatValue, FormattedCryptoAmount, Translation } from 'src/components/suite';
+import { Paragraph, Radio } from '@trezor/components';
+import { spacingsPx } from '@trezor/theme';
+import { Inputs } from './Inputs';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { mapTestnetSymbol } from 'src/utils/wallet/coinmarket/coinmarketUtils';
+import { useSelector } from 'src/hooks/suite';
+import { useUnstakeEthFormContext } from 'src/hooks/wallet/useUnstakeEthForm';
+import { selectSelectedAccountEverstakeStakingPool } from 'src/reducers/wallet/selectedAccountReducer';
+
+const GreyP = styled(Paragraph)`
+    color: ${({ theme }) => theme.textSubdued};
+`;
+
+const StyledRadioButton = styled(Radio)`
+    align-items: center;
+
+    & > div:nth-child(3) {
+        padding: 14px 0;
+        margin-left: ${spacingsPx.xxs};
+        border-bottom: 1px solid ${({ theme }) => theme.borderOnElevation1};
+        flex: 1 0 auto;
+    }
+
+    &:nth-child(3) {
+        & > div:nth-child(3) {
+            border-bottom: none;
+        }
+    }
+`;
```

Comment:
This should be in the wrapper, please do not override components from the design system 🙏 . It's a bad pattern and we are going to avoid it in the future.